jscomp/bsb/templates/react-examples/webpack.config.js

```diff
@@ -0,0 +1,13 @@
+const path = require('path');
+
+module.exports = {
+  entry: {
+    async: './lib/js/src/async/counterRoot.js',
+    simple: './lib/js/src/simple/simpleRoot.js',
```

Comment:
should this be changed accordingly?

Reply:
I haven't made any change to "react-examples", since it's meant to mirror the reason-react-example repo. I just renamed it from what was previously "react".